app/backend/articles/urls.py

```diff
@@ -0,0 +1,9 @@
+from articles import views
+from rest_framework.authtoken.views import obtain_auth_token 
+from django.urls import path, include
+
+urlpatterns = [
+    path('all', views.get_all_articles, name='get_all_articles'),
```

Comment:
- "/all" may change as "/articles". 
- In the Forum endpoint we chose "/post" as *create post* url; thus, "/article" is a consistent application.

Reply:
* it is "/articles/all" already.
* Yes, it is.

Reply:
> * it is "/articles/all" already.

Oh, right...